install_config/build_defaults_overrides.adoc

```diff
@@ -0,0 +1,109 @@
+= Configuring Global Build Defaults and Overrides
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Developers can define settings in specific build configurations within their
+projects, such as
+link:../dev_guide/builds.html#using-a-proxy-for-git-cloning[configuring a proxy
+for Git cloning]. Rather than requiring developers to define certain settings in
+each of their build configurations, cluster administrators can use admission
+control plug-ins to configure global build defaults and overrides that
+automatically populate these settings into any build configuration.
```

Comment:
Fixed per above suggestion, and added this paragraph that follows for clarification:

> The settings from these plug-ins are not set in the build configurations or
builds themselves, but rather are only used during the build process. This
allows administrators to change the global configuration at any time, and any
builds that are re-run from existing build configurations or builds will get the
new settings.